src/main/java/com/github/fabienrenaud/jjb/databind/Deserialization.java

```diff
@@ -15,6 +17,16 @@
  * @author Fabien Renaud
  */
 public class Deserialization extends JsonBench {
+    static final ObjectReaderProvider featuresProvider = new ObjectReaderProvider();
+    static final JSONReader.Context featuresContext;
+    static {
+        featuresProvider.setDisableArrayMapping(true);
```

Comment:
I cannot find such methods when searching on github / fastjson2. Can you provide a link to show me where these are published/pushed?